### ISSUE TITLE
"Open as link" in Quick Menu does not work since 1.27

### ISSUE THREAD

**Issue by Fanolian** [ssborbis/ContextSearch-web-ext#292]
### Steps to reproduce
1. In Firefox, install ContextSearch-web-ext 1.27 to 1.27.2.
2. highlight a link. Quick link: `https://www.google.com`.
3. Open Quick Menu.
4. Click/middle click "Open as link".

### Actual result
Nothing happens. The link does not open.
In Nightly, Web Console shows this error when I click the "Open as link" button:
````
21:27:19.484 Uncaught (in promise) Error: can't access property "type", node is null
    s moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    s moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    apply moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    init moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tools.js:75
    addTileEventHandlers moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:149
    addTileEventHandlers moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:118
    init moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tools.js:71
    createToolsArray moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:578
    createToolsArray moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:570
    makeQuickMenu moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:667
    makeFrameContents moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/quickmenu.js:26
    <anonymous> moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/quickmenu.js:291
    async* moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/quickmenu.js:281
undefined
    r moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    r self-hosted:1208
    apply self-hosted:2644
    applySafeWithoutClone resource://gre/modules/ExtensionCommon.jsm:613
    wrapPromise resource://gre/modules/ExtensionCommon.jsm:842
    withLastError resource://gre/modules/ExtensionCommon.jsm:761
    wrapPromise resource://gre/modules/ExtensionCommon.jsm:830
    (Async: promise callback)
    wrapPromise resource://gre/modules/ExtensionCommon.jsm:812
    callAsyncFunction resource://gre/modules/ExtensionCommon.jsm:1090
    callAsyncFunction resource://gre/modules/ExtensionChild.jsm:720
    callAndLog resource://gre/modules/ExtensionChild.jsm:696
    callAsyncFunction resource://gre/modules/ExtensionChild.jsm:719
    stub resource://gre/modules/Schemas.jsm:2690
    s moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    s moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    s self-hosted:1265
    apply moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/lib/browser-polyfill.min.js:1
    init moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tools.js:75
    addTileEventHandlers moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:149
    (Async: EventListener.handleEvent)
    addTileEventHandlers moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:118
    init moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tools.js:71
    createToolsArray moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:578
    forEach self-hosted:206
    createToolsArray moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:570
    makeQuickMenu moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/tilemenu.js:667
    makeFrameContents moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/quickmenu.js:26
    <anonymous> moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/quickmenu.js:291
    AsyncFunctionNext self-hosted:674
    (Async: async)
    <anonymous> moz-extension://47f14e71-7298-4178-8952-5db63c2bc0e3/quickmenu.js:281
````

### Notes
Tested on Firefox 81.0.2 and Nightly 2020-10-18 build.
v1.26.1 works but not v1.27+.

Thanks.

**Comment by ssborbis** [comment]
Thanks! I've got it fixed now. Hotfix going up to AMO shortly

**Comment by alexolog** [comment]
Getting empty pages when trying to highlight and search on thesaurus.com
Is this the same issue?

**Comment by Fanolian** [comment]
Fixed in 1.27.3. Thanks.
@alexolog If you issue hasn't been fixed in 1.27.4. Please open another report. Thanks.

**Comment by alexolog** [comment]
Seems to be working now,  Thanks!
